src/styles/blocks/header.scss

```
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: var(--animation-duration-header);
```

```suggestion
    transition: color var(--animation-duration-header);
```
Transition properties are still not specified. Please fix it like that
https://developer.mozilla.org/en-US/docs/Web/CSS/transition